pages/common/csvtool.md

```diff
@@ -0,0 +1,24 @@
+# csvtool
+
+> Utility to filter and extract data from CSV formatted sources.
+> More information: <https://github.com/maroofi/csvtool/>.
+
+- Extract 2nd column from example.csv:
+
+`csvtool -c 2 test.csv`
```

Comment:
We use a path to file, and long argument if is available
```suggestion
`csvtool --column 2 {{path/to/file.csv}}`
```

Reply:
Fixed, thanks the the feedback.